kotlin/src/main/kotlin/org/axonframework/extensions/kotlin/CommandGatewayExtensions.kt

```diff
@@ -19,43 +19,28 @@
  */
 package org.axonframework.extensions.kotlin
 
-import org.axonframework.commandhandling.CommandCallback
 import org.axonframework.commandhandling.CommandMessage
 import org.axonframework.commandhandling.gateway.CommandGateway
 import org.axonframework.messaging.MetaData
 import java.util.concurrent.TimeUnit
 
 /**
- * Extension for command gateway.
+ * Callback-style send with dedicated on-success and on-error functions (defaults do nothing)
```

Comment:
This is still an extension function of the `CommandGateway` it's `send(C, CommandCallback<? super C, ? super R>)`, right? I think the documentation should specify that, as well as linking to the original method's documentation with the bracket (`[CommandGateway.send]` notation.